### ISSUE TITLE
Simplify converter

### ISSUE THREAD

**Issue by mipastgt** [HanSolo/SVGPathConverter#1]
The converter could be simplified by directly using the relative and absolute settings of path elements like this.

        MoveTo mt = new MoveTo(10, 20);
        mt.setAbsolute(false);

This would allow for a more direct mapping of the original SVG and you only would have to maintain one implementation for relative and absolute placement of elements.

**Comment by HanSolo** [comment]
Yep that’s correct…did not thought about that, thx for the hint 👍🏻
